var/spack/repos/builtin/packages/geos/package.py

```diff
@@ -40,17 +43,15 @@ class Geos(AutotoolsPackage):
     variant('python', default=False, description='Enable Python support')
 
     extends('ruby', when='+ruby')
+    extends('python', when='+python')
 
     # Python 3 is supposedly supported, but I couldn't get it to work
     # https://trac.osgeo.org/geos/ticket/774
-    extends('python@:2', when='+python')
+    depends_on('python@:2', when='@:3.5')
 
     depends_on('swig', type='build', when='+ruby')
     depends_on('swig', type='build', when='+python')
 
-    # `make check` fails with:
-    # FAIL: geos_unit
```

Comment:
The unit tests no longer fail in the latest release, yay!

Reply:
..but shouldn't we keep this comment in here, for releases up through 3.6.2?

Reply:
Meh, I'm the only one who runs unit tests anyway.